src/audio/pipeline.c

```diff
@@ -63,6 +63,7 @@ void pipeline_posn_init(struct sof *sof)
 }
 
 static enum task_state pipeline_task(void *arg);
+static void pipeline_schedule_cancel(struct pipeline *p);
```

Comment:
Are we 100% sure we won't need this function to be exposed in the future? But I mean the change can be easily reverted if that happens...

Reply:
Yes, I think we can revert if needed.